packages/bruno-app/src/utils/common/index.js

```diff
@@ -76,7 +76,7 @@ export const normalizeFileName = (name) => {
     return name;
   }
 
-  const validChars = /[^\w\s-]/g;
+  const validChars = /[^\w\s\u0430-\u044f-]/gi;
```

Comment:
As far as I understand, this is only add Cyryllic to basic alphanumeric characters.  That still leaves quite a number of characters in other scripts as well as diacritics in Latin, that should be accepted based on the same principle. On top of that I'd argue the `\s` excludes a few problematic variants of whitespace that **should** in fact be escaped.

My proposition is the following:
```/[^\p{L}\p{N} \.-]/gu```
That is:
-  `/gu` for full unicode
- `\p{L}` - any kind of letter from any script
- `\p{N}` - any kind of numeric character in any script
- ` \._-` - literal whitespace, a dot, a dash and underscore

Reply:
I agree with expanding that regex to exclude more characters. Also technically that regex variable should probably be called `invalidChars`.